test/Table_Tests/src/Common_Table_Operations/Filter_Spec.enso

```diff
@@ -434,6 +434,23 @@ spec setup =
             err2 = t2.filter "Y" (Filter_Condition.Not_Equal 5) on_problems=Problem_Behavior.Report_Error
             err2.should_fail_with Floating_Point_Equality
 
+        Test.specify "should attach a warning when Nothing is used as a value in a comparison or `is_in` `Filter_Condition`" <|
+            t = table_builder [["x", [1, 2, 3, 4, 5]]]
+            fcs = [Filter_Condition.Equal Nothing, Filter_Condition.Not_Equal Nothing]
+                + [Filter_Condition.Less Nothing, Filter_Condition.Equal_Or_Less Nothing]
+                + [Filter_Condition.Equal_Or_Greater Nothing, Filter_Condition.Greater Nothing]
+                + [Filter_Condition.Between Nothing Nothing , Filter_Condition.Is_In [Nothing]]
+                + [Filter_Condition.Not_In [Nothing]]
+                + [Filter_Condition.Is_In [1, Nothing, 2]]
```

Comment:
Please add a test for the same cases as here but having a column instead of the vector.

I imagine there is no significant difference between the two, right? A vector containing Nothing or column containing Nothing will yield the same results for `Is_In`, so IMO the warnings should also be aligned between the two.

Reply:
Added column tests.